charts/newrelic-infrastructure/README.md

```diff
@@ -30,7 +30,7 @@ This chart will deploy the New Relic Infrastructure agent as a Daemonset.
 | `image.pullPolicy`             | The pull policy.                                                                                                                                                                                                                                  | `IfNotPresent`                  |
 | `image.pullSecrets`            | Image pull secrets.                                                                                                                                                                                                                               | `nil`                           |
 | `image.tag`                    | The version of the container to pull.                                                                                                                                                                                                             | `2.0.0`                       |
-| `image.windowsTag`             | The version of the Windows container to pull.                                                                                                                                                                                                     | `1.21.0-windows-1809-alpha`     |
+| `image.windowsTag`             | (Deprecated) The version of the Windows container to pull.                                                                                                                                                                                                     | `1.21.0-windows-1809-alpha`     |
```

Comment:
Can we add a `NOTES.txt` that yells at the user if this property is defined? I fear the deprecation might not have enough visibility otherwise.

Reply:
Sure, it makes sense, I add that